src/client/package.json

```diff
@@ -18,6 +18,7 @@
     "@expo/metro-runtime": "~3.2.3",
     "@react-native-community/datetimepicker": "^8.2.0",
     "@react-native-community/slider": "^4.5.5",
+    "@react-native-picker/picker": "^2.9.0",
```

Comment:
can we also update the husky prepare command, we could probs just comment it or remove it for now